security/pod-to-pod-encryption.md

```diff
@@ -1,14 +1,11 @@
 ---
-title: Try out the pod-to-pod encryption tech preview
-description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+title: Pod to pod encryption
```

Comment:
Suggest "Encrypt in-cluster pod traffic".  Might also change the path to be `security/encrypt-pod-traffic` or similar (which isn't quite as accurate but will likely score better with Google SEO).